tests/integration/multicluster/main_test.go

```diff
@@ -0,0 +1,54 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package multicluster
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	ist istio.Instance
+	g   galley.Instance
+	p   pilot.Instance
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("multicluster", m).
+		Label(label.Multicluster).
+		RequireEnvironment(environment.Kube).
+		RequireClusters(2).
+		SetupOnEnv(environment.Kube, istio.Setup(&ist, nil)). // By default control plane is installed in cluster[0]
```

Comment:
I thought this was replicated, shouldn't it be installed in all clusters?

Reply:
Right, replicated is the default and we don't change it here. The clusters are extracted from the environment within the setup fn.

Reply:
I don't understand the purpose of the comment here. The test doesn't specific a topology here.